### ISSUE TITLE
Final Paper Suggestion

### ISSUE THREAD

**Issue by cerdamejiaj** [burrowss/burrowss.github.io#7]
Your article was overall great, just minor changes and you will be set
**Paragraph 2**
“What does net neutrality mean to me”.  put the period before quotations 
 
"Any user of the Internet is guaranteed access to any legal portion of the Internet with uniform priority" Try re-wording this sentence is a bit confusing. Maybe changing " legal portion " to something else.

"This means that ISPs are not legally allowed to charge more for any certain type of data that a user may use. " I think you should say "charge more for any type of content or data"

**Paragraph 3**
"This idea however is incredibly flawed" change it to "This idea, however,... "
"result in ISPs charging more to video services such " remove the to and add fo. "more for"

The phrase in order to may be wordy. Consider just having "to keep their"

"This increase in cost to the content providers would force them to increase the price of their product"
consider rewriting this sentence, it a bit wordy and hard to get you point.

The word large is often overused --> "This possibility would result in large businesses" change to "in great businesses paying"

"between massive companies " delete massive and use giant or huge

**Paragraph 5**
to truly get the voice of the people heard. reword this to --> "to get the voice of the people heard truly"

The word key is often overused. "awareness of this issue is key in ensuring that" use essential or critical

**Paragraph 6** 
"where anyone and everyone is entitled to " delete is to are

" as the Republican party " put a comma after party

**Comment by burrowss** [comment]
Thanks for all the suggestion!

**Comment by burrowss** [comment]
Made some changes. If you see anything else please let me know
